owid/catalog/catalogs.py

```diff
@@ -63,13 +67,15 @@ class LocalCatalog(CatalogMixin):
     """
 
     path: Path
+    frame: "CatalogFrame"
 
     def __init__(self, path: Union[str, Path]) -> None:
         self.path = Path(path)
-        if self._catalog_file.exists():
-            self.frame = CatalogFrame(pd.read_feather(self._catalog_file.as_posix()))
-        else:
-            self.frame = CatalogFrame.create_empty()
```

Comment:
Good decision on not supporting empty catalogs, as there would not be a `base_uri`

Reply:
Yeah, it was not very helpful to have an empty catalog here 😅